### PR TITLE
LB-1111 : Cover art not shown on recommendation pages

### PR DESCRIPTION
**Pull request by jivteshsingh**
# Problem

  LB-1111 :  Cover art not shown on recommendation pages 
         
# Solution

The "caa_id" and  "caa_release_mbid" and "release_mbid" were missing from the recommendations metadata which was creating problem in rendering the covert art in frontend.

# Action

Added the "caa_id" and  "caa_release_mbid"  and "release_mbid" which resulted in generating the cover art image and rendering it in frontend and fixing the bug.

### CODE REVIEW

File: listenbrainz/webserver/views/recommendations_cf_recording.py [hunk 1]
```diff
@@ -133,6 +133,25 @@ def _get_playable_recommendations_list(mbids_and_ratings_list):
         Args:
             mbids_and_ratings_list: Contains recording mbid and corresponding score.
 
+
+
+
+            if caa_id and caa_release_mbid or record_mbid are avialable then:
```
Comment (MonkeyDo) [inline listenbrainz/webserver/views/recommendations_cf_recording.py:139]:
```suggestion
            if caa_id and caa_release_mbid or recording_mbid are available then:
```

File: listenbrainz/webserver/views/recommendations_cf_recording.py [hunk 2]
```diff
@@ -133,6 +133,25 @@ def _get_playable_recommendations_list(mbids_and_ratings_list):
...
+            Returns:
+                'track_metadata' : {
+                    'additional_info' : {
+                        'caa_id' : "demo caa id",
+                        'caa_release_mbid' : "demo caa release album mbid"
+                        'release_mbid' : "demo release mbid"
+                    }
+                }
+
+                That means to render cover art caa id to be present in metadata is compulsory otherwise it will render default image in frontend.
```
Comment (MonkeyDo) [inline listenbrainz/webserver/views/recommendations_cf_recording.py:151]: Not sure I understand this sentence; can you rephrase it?

**Review by MonkeyDo — COMMENTED** [review]
Nice job!
Tested and working on test.LB:
image

File: listenbrainz/webserver/views/recommendations_cf_recording.py (hunk 2 above)
Comment (jivteshsingh) [reply listenbrainz/webserver/views/recommendations_cf_recording.py:151]: Sure I will rephrase it and thanks for reviewing it.

File: listenbrainz/webserver/views/recommendations_cf_recording.py [hunk 3]
```diff
@@ -133,6 +133,25 @@ def _get_playable_recommendations_list(mbids_and_ratings_list):
...
+            Returns:
+                'track_metadata' : {
+                    'additional_info' : {
+                        'caa_id' : "demo caa id",
+                        'caa_release_mbid' : "demo caa release album mbid"
+                        'release_mbid' : "demo release mbid"
+                    }
+                }
+
+                These changes are done because to show cover art of a particular listen its recording_mbid and then its release_mbid, caa_id , caa_release_mbid are needed in order to perform the frontend logic in utils.tsx and render it on frontend of the web app.
```
Comment (MonkeyDo) [inline listenbrainz/webserver/views/recommendations_cf_recording.py:151]: OK, now I understand what you meant, thank you.
I think this is too verbose as it is; I suggest the following which should be enough for this docstring:

```suggestion
                These fields are required to show cover art on the front-end web app.
```

Comment (jivteshsingh) [reply listenbrainz/webserver/views/recommendations_cf_recording.py]: updated docstring

**Review by MonkeyDo — APPROVED** [review]
Thank you !
